src/index.jsx

```diff
@@ -1,22 +1 @@
-import { render } from 'react-dom';
-import { AppContainer } from 'react-hot-loader';
-import { BrowserRouter } from 'react-router-dom';
-import App from './App';
-
-const root = document.getElementById('root');
-const load = () => render(
-  (
-    <AppContainer>
-      <BrowserRouter>
-        <App />
-      </BrowserRouter>
-    </AppContainer>
-  ), root,
-);
-
-// This is needed for Hot Module Replacement
-if (module.hot) {
-  module.hot.accept('./App', load);
-}
-
-load();
+export default () => 'Welcome to perf-goggles';
```

Comment:
It doesn't look like this library does anything.

Reply:
Would including the perfherder.js module and exporting it be the right way?

Something like what you do [here](https://github.com/taskcluster/taskcluster-client-web/blob/master/src/index.js)?